src/Rings/orderings.jl

```diff
@@ -1218,6 +1218,33 @@ function is_mixed(ord::MonomialOrdering)
   return false
 end
 
+@doc Markdown.doc"""
+    cmp(ord::MonomialOrdering, a::MPolyElem, b::MPolyElem)
+
+Compare monomials `a` and `b` with the ordering `ord`: Return `-1` for `a < b`
+and `1` for `a > b`. A return of `0` indicates that either `a == b` or `ord` is
+a partial ordering that does not distinguish `a` from `b`.
```

Comment:
@thofma Is this juliaesque enough?

Reply:
How useful would it if it throws in case `a` and `b` are not comparable? I don't know the application, so just asking.

Reply:
Clearly is it less useful if it functions in strict subsets of the cases. But, making functions less useful puts less of a burden on the developers, so I'm going to do it.